app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -257,8 +254,8 @@ public void typeMode() {
         typeMode = true;
     }
 
-    public void speakMode() {
-        Log.d(DEBUG_TAG, "enabled speakMode");
+    public void doneTypeMode() {
```

Comment:
Because I had a hard time understanding/remembering what "speakMode" meant. Really, the method is called whenever you're done typing, so I named it as such (and you're not actually speaking.)